pandas/core/window.py

```diff
@@ -1382,17 +1354,18 @@ class Expanding(_Rolling_and_Expanding):
     By default, the result is set to the right edge of the window. This can be
     changed to the center of the window by setting ``center=True``.
 
-    The `freq` keyword is used to conform time series data to a specified
-    frequency by resampling the data. This is done with the default parameters
-    of :meth:`~pandas.Series.resample` (i.e. using the `mean`).
+    See Also
+    ---------
```

Comment:
I think these need to be the same length as the text (the underlines)

Reply:
Actually, it has to be the same length or longer. But yeah, it's probably better style to keep same length. I'll change it.